api/entities/config.js

```diff
@@ -94,10 +97,11 @@ class Config {
         if (this.name && this.name.length <= 3) deferred.reject('Config\'s name should be longer then 3 symbols');
         if (typeof(this.enabled) !== 'boolean') deferred.reject('Enabled field should be boolean');
         if (!this.isConfigValid(this.config)) deferred.reject('Enabled field should be boolean');
+        if (!this.priority) deferred.reject('Config couldn\'t be without priority');
+        if (!PriorityTypes.getTypes().find((type) => type.id === this.priority)) deferred.reject('Wrong priority type');
```

Comment:
I see three methods here: `validate()`, `validateName()` and `validatePriorityType()`. Could it be?